plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -508,6 +508,7 @@ public void testTargetMaxFileSize()
 
         Session session = Session.builder(getSession())
                 .setSystemProperty("task_writer_count", "1")
+                .setSystemProperty("task_scale_writers_enabled", "false")
```

Comment:
Can we keep this enabled (here and in other places)? It's always confusing if we improve something but then change default in tests to pre-optimization value.

Reply:
It is essential to disable it at these places because all these tests are testing directly/indirectly the number of files or file sizes with different values of `task_writer_count`. So, enabling it could make them flaky or incorrect.

Reply:
Add a comment then please